src/cli/config-flags.ts

```diff
@@ -213,13 +213,25 @@ export const CLI_FLAG_ALIASES: AliasMap = {
   h: 'help',
   p: 'port',
   v: 'version',
+
+  // JEST SPECIFIC CLI FLAGS
+  // these are defined in
+  // https://github.com/facebook/jest/blob/4156f86/packages/jest-cli/src/args.ts
+  b: 'bail',
+  e: 'expand',
+  w: 'maxWorkers',
+  o: 'onlyChanged',
+  f: 'onlyFailures',
+  i: 'runInBand',
+  t: 'testNamePattern',
+  u: 'updateSnapshot',
 };
 
 /**
  * A regular expression which can be used to match a CLI flag for one of our
  * short aliases.
  */
-export const CLI_FLAG_REGEX = new RegExp(`^-[chpv]{1}$`);
+export const CLI_FLAG_REGEX = new RegExp(`^-[chpvbewofitu]{1}$`);
```

Comment:
note the update to the CLI flag regex here - I dislike this, but [as noted previously](https://github.com/ionic-team/stencil/pull/3765#discussion_r1009465356), the obvious thing you'd like to do here is the following:


```suggestion
export const CLI_FLAG_REGEX = new RegExp(`^-[${
  Object.keys(CLI_FLAG_ALIASES).join("")
}]{1}$`);
```

but unfortunately that causes some tree-shaking issues because (I speculate) Rollup can't determine that if nothing imports this value it will be Ok to remove both it and `CLI_FLAG_ALIASES` from the bundle. Bah!